theseus/geometry/so3.py

```diff
@@ -185,12 +185,12 @@ def exp_map(tangent_vector: torch.Tensor) -> "SO3":
         return ret
 
     def _log_map_impl(self) -> torch.Tensor:
-        ret = torch.zeros(self.shape[0], 3, dtype=self.dtype, device=self.device)
-        ret[:, 0] = 0.5 * (self[:, 2, 1] - self[:, 1, 2])
-        ret[:, 1] = 0.5 * (self[:, 0, 2] - self[:, 2, 0])
-        ret[:, 2] = 0.5 * (self[:, 1, 0] - self[:, 0, 1])
+        sine_axis = torch.zeros(self.shape[0], 3, dtype=self.dtype, device=self.device)
```

Comment:
We should also add unit tests that will fail with the previous version (something that checks the gradient).

Reply:
Will be added in another PR.